pkgs/development/python-modules/torchvision/binary-hashes.nix

```diff
@@ -3,20 +3,41 @@
 # Precompiled wheels can be found at:
 # https://download.pytorch.org/whl/torch_stable.html
 
-version: {
-  x86_64-linux-37 = {
-    name = "torchvision-${version}-cp37-cp37m-linux_x86_64.whl";
-    url = "https://download.pytorch.org/whl/cu111/torchvision-${version}%2Bcu111-cp37-cp37m-linux_x86_64.whl";
-    hash = "sha256-7EMVB8KZg2I3P4RqnIVk/7OOAPA1OWOipns58cSCUrw=";
-  };
-  x86_64-linux-38 = {
-    name = "torchvision-${version}-cp38-cp38-linux_x86_64.whl";
-    url = "https://download.pytorch.org/whl/cu111/torchvision-${version}%2Bcu111-cp38-cp38-linux_x86_64.whl";
-    hash = "sha256-VjsCBW9Lusr4aDQLqaFh5dpV/5ZJ5PDs7nY4CbCHDTA=";
+# To add a new version, run "prefetch.sh 'new-version'" to paste the generated file as follows.
+
+version : builtins.getAttr version {
+  "0.9.1" = {
+    x86_64-linux-37 = {
+      name = "torchvision-0.9.1-cp37-cp37m-linux_x86_64.whl";
+      url = "https://download.pytorch.org/whl/cu111/torchvision-0.9.1%2Bcu111-cp37-cp37m-linux_x86_64.whl";
+      hash = "sha256-7EMVB8KZg2I3P4RqnIVk/7OOAPA1OWOipns58cSCUrw=";
+    };
+    x86_64-linux-38 = {
+      name = "torchvision-0.9.1-cp38-cp38-linux_x86_64.whl";
+      url = "https://download.pytorch.org/whl/cu111/torchvision-0.9.1%2Bcu111-cp38-cp38-linux_x86_64.whl";
+      hash = "sha256-VjsCBW9Lusr4aDQLqaFh5dpV/5ZJ5PDs7nY4CbCHDTA=";
+    };
+    x86_64-linux-39 = {
+      name = "torchvision-0.9.1-cp39-cp39-linux_x86_64.whl";
+      url = "https://download.pytorch.org/whl/cu111/torchvision-0.9.1%2Bcu111-cp39-cp39-linux_x86_64.whl";
+      hash = "sha256-pzR7TBE+WcAmozskoeOVBuMkGJf9tvsaXsUkTcu86N8=";
+    };
   };
-  x86_64-linux-39 = {
-    name = "torchvision-${version}-cp39-cp39-linux_x86_64.whl";
-    url = "https://download.pytorch.org/whl/cu111/torchvision-${version}%2Bcu111-cp39-cp39-linux_x86_64.whl";
-    hash = "sha256-pzR7TBE+WcAmozskoeOVBuMkGJf9tvsaXsUkTcu86N8=";
+  "0.10.0" = {
+    x86_64-linux-37 = {
+      name = "torchvision-0.10.0-cp37-cp37m-linux_x86_64.whl";
+      url = "https://download.pytorch.org/whl/cu111/torchvision-0.10.0%2Bcu111-cp37-cp37m-linux_x86_64.whl";
+      sha256 = "1kh6zrid4jv2qh1fpgnghsy2p0k5dc9pmk90484bdziskn313j68";
```

Comment:
It would be nice to be consistent between `hash`/`sha256`. I believe that `hash` is preferred over `sha256`.

Reply:
pytorch has lts(1.8.x) and latest one(1.9.x). 
Enterprise application may use lts-version.
General developers use  latest one.
0.9.1 version is for pytorch-1.8.x.

Reply:
@samuela 
Thank you for your reviewing!
Changed `sha256` to `hash`.

Reply:
I think we should only keep the latest one. The general policy for Python packages in nixpkgs is that we do not ship multiple versions. Also, as long as there are no packages that depend on older versions, there is no real need. If someone needs the LTS version, they can always make their own copy of the derivation or override it.

Reply:
If I'm understanding @junjihashimoto correctly, I think the issue is that we currently have a pytorch LTS version packaged in nixpkgs with some users, and for those users torchvision-bin must be packaged at 0.9.1.

Reply:
We have PyTorch 1.9.0:

https://github.com/NixOS/nixpkgs/blob/a3bf844c4781244c989c3367192c8b62eff50431/pkgs/development/python-modules/pytorch/bin.nix#L21
https://github.com/NixOS/nixpkgs/blob/a3bf844c4781244c989c3367192c8b62eff50431/pkgs/development/python-modules/pytorch/default.nix#L120

Reply:
Oh well in that case no point keeping around the old version

Reply:
For now, I revert it. It looks better.

I understand that it is different from the overall python-package-management.
On the other hand, we continue to support multiple runtime versions not latest one only.